frontend/src/components/List/Column/index.ts

```diff
@@ -33,3 +33,12 @@ export * from './BlockTimeColumn';
 export * from './BlockPropagationColumn';
 export * from './LastBlockColumn';
 export * from './UptimeColumn';
+export * from './CpuArchitectureColumn'; //extra columns added
```

Comment:
Dq: but given new column follow the same structure and same pattern, can't they be factored out into one?

Reply:
Yeah I'm sure we could simplify this stuff a bunch!

Reply:
Agreed